### PR TITLE
Create GitHub action to build and publish package on PyPI

### LINKED ISSUE #92: Automate PyPI package deployment upon release

### PR DESCRIPTION
**Pull request by pierrepo**
Fix #92

The PR aims to automate the creation of the Python package and its publication on PyPI after each release.

### CODE REVIEW

**Comment by pierrepo** [comment]
For some reason, the CI is failing :cry:

**Comment by pierrepo** [comment]
@HubLot could you please have a look?

**Comment by HubLot** [comment]
yes, i'll look at it

**Comment by HubLot** [comment]
looks like it's working now. I just re-launch the tests :D

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install build
+    - name: Build package
+      run: |
+        python python -m build
```
Comment (HubLot) [inline .github/workflows/publish-to-pypi.yml:28]: remove one `python`

Comment (pierrepo) [reply .github/workflows/publish-to-pypi.yml]: oups. Thanks @HubLot

**Comment by patrickfuchs** [comment]
So are we ready for the merge?

**Comment by pierrepo** [comment]
Yes!

**Pull request merged**